spec/helpers/jwt_auth_helpers_spec.rb

```diff
@@ -0,0 +1,112 @@
+require 'rails_helper'
+
+RSpec.describe JWTAuthHelpers, type: :helper do
+  describe '#user_info_from_header' do
+    it 'should return nil when the request does not have an Authorization header' do
+      allow(helper).to receive(:headers).and_return({})
+      expect(helper.send(:user_info_from_header)).to eq(nil)
```

Comment:
It is generally considered bad practice to test private methods. Testing the public API is usually sufficient.

http://stackoverflow.com/questions/16197332/should-i-test-private-methods-using-rspec has some discussion on this.